src/main/fc/fc_rc.c

```diff
@@ -179,42 +188,19 @@ static void checkForThrottleErrorResetState(uint16_t rxRefreshRate)
 
 void processRcCommand(void)
 {
-    static float rcCommandInterp[4] = { 0, 0, 0, 0 };
```

Comment:
Cant believe all of this was being set each time the loop ran.

Reply:
kidBrazil it looks like you didn't notice the static. Something like this is in plenty of functions and other firmwares. The set doesn't happen with every loop